flags.go

```diff
@@ -234,6 +252,13 @@ func newApp() (app *cli.App) {
 					"This is effective when --disable-http2 is set.",
 			},
 
+			cli.IntFlag{
+				Name:  "max-idle-conns-per-host",
+				Value: 100,
+				Usage: "The number of MaxIdleConnsPerHost parameter in " +
```

Comment:
this comment doesn't look correct. please check.

Reply:
fixed